ci/main.tf

```diff
@@ -0,0 +1,242 @@
+#########################################
```

Comment:
I see some high level things you'll want to verify / dive deeper into with @anl , @chris-esterlein and / or @griffinfujioka :

- I don't see a backend defined which I imagine means currently will only use a local backend to manage the terraform state (ie. your laptop). Could be wrong but I believe you'll need to define a remote backend (SW team uses GCS as a backend I believe)
- The AWS "default" VPC should really never be used. You want to be using one of the custom VPCs that we have setup. This configuration will be different whether you're deploying to the Dev account or the Prod account.
- Not sure where you've been testing this, but as mentioned above, we have a Dev and a Prod account for which some configurations may vary: specifically VPC / AZ and potentially security group settings. To manage different environment targets effectively, you'll want to introduce terraform var files per environment
- I'm personally a fan of having the terraform code in the same repo, however the current pattern we have is setup with a monorepo (on bitbucket for now) -> https://aicsbitbucket.corp.alleninstitute.org/projects/SW/repos/terraform-platform/browse. The repo has an automatic plan and deploy tool (Atlantis) that has all the proper credentials and configs. I believe this is the pattern we're still expected to use until we come up with something better - once again, confirm with @anl and the other devops folks.

Reply:
Thanks, this is great info.  

We don't currently have plans to deploy to prod or keep this live, for now.  It's really just meant to be up for some short bursts, demo, testing/verification at this time.  Mostly a PoC.

I've been running it in the "AICS Animated Cell" aws account (which probably should go away at some point).   So probably to get one of the custom VPCs I'll have to use my credentials on the SW account instead.  

I'm not familiar with backends in terraform so I'll have to look that one up.